virttest/utils_test/qemu/__init__.py

```diff
@@ -204,10 +204,21 @@ def _check_driver_stat():
 
     for drv in driver.split():
         error_context.context("Check %s driver state." % drv, LOG.info)
-        driver_check_cmd = (
-            r'wmic sysdriver where PathName="C:\\Windows\\System32'
-            r'\\drivers\\%s.sys" get State /value'
-        ) % drv
+        driver_svc_map = {
+            "viorng": "VirtRng",
+            "vioser": "VirtioSerial",
+            "viofs": "VirtioFsDrv",
+            "vioinput": "VirtioInput"
+        }
+        if drv in list(driver_svc_map.keys()):
+            driver_svc = driver_svc_map[drv]
+        else:
+            driver_svc = drv
+
+        driver_check_cmd = r"powershell -command"
+        driver_check_cmd += r' "Get-WmiObject Win32_SystemDriver | Where-Object'
+        driver_check_cmd += r" { $_.Name -eq '%s' }" % driver_svc
+        driver_check_cmd += r' | Select-Object state,pathname"'
```

Comment:
```suggestion
        driver_check_cmd = (
            r"powershell -command"
            r' "Get-WmiObject Win32_SystemDriver | Where-Object'
            r" { $_.Name -eq '%s' }"
            r' | Select-Object state,pathname"')  % driver_svc
```
Hi @xiagao  
I guess we can continue to use the previous method to define this cmd value, what do you think? Thanks!

Reply:
Both are okay for me.
Updated. Could you help review it?

Reply:
Also ,the result is here with latest one.
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.single_driver_install.with_vioinput.device_keyboard.virtio_win_iso_media.q35: STARTED
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.single_driver_install.with_vioinput.device_keyboard.virtio_win_iso_media.q35: PASS (457.50 s)